build_projects/dotnet-cli-build/dotnet-cli-build.csproj

```diff
@@ -16,17 +16,13 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.NETCore.App" Version="$(MicrosoftNETCoreAppPackageVersion)" />
     <PackageReference Include="Microsoft.Build" Version="15.4.8" />
+    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="15.4.8" />
     <PackageReference Include="Microsoft.CSharp" Version="4.0.1" />
     <PackageReference Include="System.Dynamic.Runtime" Version="4.0.11" />
     <PackageReference Include="System.Reflection.Metadata" Version="1.4.1-beta-24410-02" />
     <PackageReference Include="System.Runtime.Serialization.Primitives" Version="4.1.1" />
     <PackageReference Include="System.Xml.XmlSerializer" Version="4.0.11" />
     <PackageReference Include="WindowsAzure.Storage" Version="7.2.1" />
-    <PackageReference Include="NuGet.CommandLine.XPlat" Version="4.4.0-preview3-4475" />
-    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="15.4.8" />
-    <!-- This dependency was added due to an issue in restore where a lower version of this package coming from nuget.commandline.xplat
-    led to an error. This is tracked as NuGet issue : https://github.com/NuGet/Home/issues/4213 -->
```

Comment:
I could swear I already deleted these once. Perhaps it never made it in or came back in a merge.

Reply:
I looked in the history, and this dates back to when @jgoshi first converted from `project.json` to `.csproj`.  I'm thinking it was needed a long time ago and isn't needed anymore.

I checked all the `release` branches I know about, and it is in all of them.  So maybe you never made the PR?  or didn't merge it?  Either way I hope we can get rid of it here.

Reply:
I must not have sent a PR. I forget the context. I don't see any reason why these need to be kept if things are green without them.